tests/apps/courses/test_templates_course_glimpse_organization.py

```diff
@@ -0,0 +1,78 @@
+"""
+End-to-end tests for the organization acronym display in the
+course glimpse card
+"""
+
+from cms.api import add_plugin, create_page
+from cms.test_utils.testcases import CMSTestCase
+
+from richie.apps.core.helpers import create_i18n_page
+from richie.apps.courses.cms_plugins import CoursePlugin
+from richie.apps.courses.factories import CourseFactory, OrganizationFactory
+
+
+class CourseGlimpseOrganizationTestCase(CMSTestCase):
```

Comment:
Maybe keep it in the test file that tests the course glimpse in general.

Reply:
Were you talking about the `test_cms_plugins_course` ?